detector/weakcredentials/filebrowser/filebrowser.go

```diff
@@ -174,6 +174,7 @@ func checkLogin(ctx context.Context, fileBrowserIP string, fileBrowserPort int)
 	client := &http.Client{Timeout: requestTimeout}
 	targetURL := fmt.Sprintf("http://%s:%d/api/login", fileBrowserIP, fileBrowserPort)
 
+	//nolint:errchkjson // this is a static struct, so it cannot fail
```

Comment:
@cuixq just wanting to double check I've understood your comment [here](https://github.com/google/osv-scalibr/issues/455#issuecomment-2657966744) correctly - is this now allowed by the internal linters, or should I put it on a dedicated line?